portal_pages/forms.py

```diff
@@ -1,10 +1,77 @@
-from django.forms import ModelForm
+from datetime import datetime
+
+from django import forms
+from django.forms import ModelForm, widgets
```

Comment:
It's a bit odd to have this in addition to the "from django import forms". Usually if you have the first you would just use forms.ModelForm, forms.widgets instead of importing these independently. Alternatively import explicitly whatever was added to use forms.<whatever>...